contracts/CompoundGovernor.sol

```diff
@@ -135,32 +135,75 @@ contract CompoundGovernor is
         return GovernorSequentialProposalIdUpgradeable.hashProposal(_targets, _values, _calldatas, _descriptionHash);
     }
 
+    /// @notice Creates a new proposal.
+    /// @dev This is an override of the function that skips proposal threshold check for whitelisted accounts.
+    /// @param _targets An array of addresses that will be called if the proposal is executed.
+    /// @param _values An array of ETH values to be sent to each address when the proposal is executed.
+    /// @param _calldatas An array of calldata to be sent to each address when the proposal is executed.
+    /// @param _description A human-readable description of the proposal.
+    /// @return uint256 The ID of the newly created proposal.
+    function propose(
+        address[] memory _targets,
+        uint256[] memory _values,
+        bytes[] memory _calldatas,
+        string memory _description
+    ) public override(GovernorUpgradeable) returns (uint256) {
+        address _proposer = _msgSender();
+
+        // check description restriction
+        if (!_isValidDescriptionForProposer(_proposer, _description)) {
+            revert GovernorRestrictedProposer(_proposer);
+        }
+
+        if (!isWhitelisted(_proposer)) {
+            // check proposal threshold
+            uint256 _votesThreshold = proposalThreshold();
+            if (_votesThreshold > 0) {
+                uint256 _proposerVotes = getVotes(_proposer, clock() - 1);
+                if (_proposerVotes < _votesThreshold) {
+                    revert GovernorInsufficientProposerVotes(_proposer, _proposerVotes, _votesThreshold);
+                }
+            }
+            return _propose(_targets, _values, _calldatas, _description, _proposer);
+        } else {
```

Comment:
The "return" statements are exactly the same, so with this refactoring, no need for an "else" clause of the "If" here, with the "return" statement at the bottom of the function. A small code savings.